core/transaction_test.go

```diff
@@ -251,6 +257,27 @@ func TestDeclareTransaction(t *testing.T) {
 			if !transactionHash.Equal(test.want) {
 				t.Errorf("wrong hash: got %s, want %s", transactionHash.Text(16), test.want.Text(16))
 			}
+
+			CheckTransactionSymmetry(t, test.input)
 		})
 	}
 }
+
+func CheckTransactionSymmetry(t *testing.T, in Transaction) {
```

Comment:
I think we should make the parameter name more descriptive. Maybe `input` or `value`? 
```suggestion
func CheckTransactionSymmetry(t *testing.T, input Transaction) {
```

Reply:
Done